<launch>

 <!-- Set non-default parameters -->

 <!--Estimation-->
  <node name="ekf" pkg="mocap_filter"  type="mocap_filter" output="screen">
    <rosparam command="load" file="$(find mocap_filter)/params/estimator.yaml" />
    <remap from="mocap" to="/naze/CG"/> 
  </node>
  
  <node name="record" pkg="rosbag" type="record" args="-O naze -a"/>
  <node name="play" pkg="rosbag" type="play" args="/home/jarvis/naze_balanced_props.bag -u 30">
   <remap from="relative_state" to="transformed_mocap"/>
  </node>
 

</launch>
